WasatchNET/EEPROM.cs

```diff
@@ -912,6 +912,28 @@ public byte matchingThreshold
         }
         byte _matchingThreshold = 90;
 
+        public byte throwAwayCount
+        {
+            get { return _throwAwayCount; }
+            set
+            {
+                _throwAwayCount = value;
+                EEPROMChanged?.Invoke(this, new EventArgs());
+            }
+        }
+        byte _throwAwayCount;
+
+        public byte untetheredMode

```

Comment:
Hi Evan, I'm realizing we now have identically-named variables "untetheredMode" in both Spectrometer and EEPROM.  The first is a flag, indicating that acquisitions should be performed in untethered mode (wValue=1, with polling before readout), while the second is a bitmask encoding several potential untethered features.  Can I request that you change the one in Spectrometer to untetheredAcquisitionEnabled, and the one in EEPROM to untetheredFeatureMask?  This way both are more clear.

Reply:
will do